src/classes/BDI_DataImport_BATCH.cls

```diff
@@ -331,7 +331,7 @@ public with sharing class BDI_DataImport_BATCH implements Database.Batchable<sOb
                     isModified = true;
                 }
             }
-            
+                        
```

Comment:
sure wish we had a build tool that would automatically convert all tabs to spaces (make Kevin happy), and remove trailing spaces like this!